doc/languages-frameworks/vim.section.md

```diff
@@ -236,12 +236,13 @@ Run the update script with a GitHub API token that has at least `public_repo` ac
 
 ```sh
 GITHUB_API_TOKEN=my_token ./pkgs/applications/editors/vim/plugins/update.py
+update
```

Comment:
can we make `update` the default subcommand?

Reply:
I guess we could (not requiring a subparser) but I am not keen of that, I feel like it would defeat the purpose of the PR to make a cleaner distinction. 
In case someone doesn't mention, the error looks descriptive enough
```
$ pkgs/applications/editors/vim/plugins/update.py
usage: update.py [-h] {add,update} ...
update.py: error: the following arguments are required: command
```

Reply:
I would prefer it to be the default since the file name is already `update.py`

Reply:
Would it make sense to split these into separate `update.py` and `add.py` scripts?

Reply:
> Would it make sense to split these into separate update.py and add.py scripts

I dont think so, it would hurt discoverability IMO. 

> I would prefer it to be the default since the file name is already update.py

if you add a plugin, you update nixpkgs too. I would prefer `updater` as a name, e.g. `pkgs/applications/editors/vim/plugins/updater` but I did not want to make this more confusing. 
At first I thought I would add subcommand-specific flags like `--unpin <PKG>` but if we update by default it might be better to have `unpin` as its own command (alternatively update --force <PKG> ? well that can be discussed in a later PR).

Reply:
I've made `update` the default and reverted the related doc changes accordingly